e2e-tests/commands/insurance/check-your-answers/complete-and-submit-multiple-check-your-answers.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * completeAndSubmitMultipleCheckYourAnswers
+ * Complete and submit multiple "Application - Check your answers" forms.
+ * @param {Integer} count: Amount of times to click the submit button.
+ */
+const completeAndSubmitMultipleCheckYourAnswers = ({ count }) => {
+  const arr = new Array(count).fill();
+
+  arr.forEach(() => {
```

Comment:
could a for loop do the same and remove the need for `const arr = new Array(count).fill();`?

Reply:
I can't see a better way to do this as it is currently.

I think a `forEach` is cleaner/simpler than a for each:

```js
arr.forEach(() => {
  cy.submitCheckYourAnswersForm();
});

for (item in arr) {
  cy.submitCheckYourAnswersForm();
}
```

RE `new Array` - `count` is a number, we need to create an array regardless.

FYI we have the same pattern in `cy.clickSubmitButtonMultipleTimes`